source/containers/Form/Form.js

```diff
@@ -61,21 +70,29 @@ const {
                   case 'start': {
                     return {
                       label: action.label,
-                      onClick: () => startForm(onStart),
+                      onClick: () => {
+                        startForm(onStart);
+                      },
                     };
                   }
 
                   case 'submit': {
                     return {
                       label: action.label,
-                      onClick: () => handleSubmit(onSubmit),
+                      onClick: () => {
+                        updateCaseInContext(formState.formAnswers, 'submitted');
+                        handleSubmit(onSubmit);
```

Comment:
ESLint: 'handleSubmit' is not defined.

Reply:
This looks different and is fixed in the newest version of this branch, so I'm saying that it's fixed.